roles/base/tasks/dotfiles.yml

```diff
@@ -20,9 +20,9 @@
       group: "{{ base_user.name }}"
 
   - name: Install dotfiles
-    shell: "bash {{ base_dotfiles_dir }}/bin/myc -i"
-    args:
-      creates: "/home/{{ base_user.name }}/.oh-my-zsh"
+    shell: "bash {{ base_dotfiles_install_cmd }}"
+    #args:
+    #  creates: "/home/{{ base_user.name }}/.oh-my-zsh"
```

Comment:
If this check is no longer present the `install_cmd` would be executed on every ansible run. We might want to generalize this by also providing a folder that is created through the dotfile installation through the args.

Alternatively we could moved the check into the command and return early with a return code if no action is necessary. Ansible could then mark the task as unchanged through [changed_when](https://docs.ansible.com/ansible/latest/user_guide/playbooks_error_handling.html#defining-changed)

Reply:
isn't that already covered by 
`
  when: not base_dotfiles_folder.stat.exists
`

this skips the whole block if e.g. the `.myconf` folder is already there.

Reply:
tested:
```
TASK [base : Debug] *******************************************************************************************************************************************************************************************************************************************************************************
ok: [ot-rpi7] => {
    "changed": false,
    "msg": [
        "User: {'name': 'flow', 'password': '$6$I4TpFltmnJ1pm$sDVMYkVxT7CglyyPA2Bq2y0SRkTS7QLZtl1DspzqNgp4R/lbrzjf.AhT3TG2IgAhan3K.7BnBzfKvdgmK9xC01', 'ssh_keys_gitlab': ['gitlab.lrz.de/Flowm'], 'dotfiles_url': 'https://github.com/Flowm/dotfiles.git', 'dotfiles_dir': '.myconf', 'dotfiles_install_cmd': 'bin/myc -i'}, Dotfiles dir is /home/flow/.myconf"
    ]
}

TASK [base : Check if dotfiles folder exists] *****************************************************************************************************************************************************************************************************************************************************
ok: [ot-rpi7]

TASK [base : Clone dotfiles] **********************************************************************************************************************************************************************************************************************************************************************
skipping: [ot-rpi7]
```